test/unit/remote/cozy.js

```diff
@@ -21,6 +21,13 @@ describe('RemoteCozy', function () {
 
   beforeEach(function () {
     remoteCozy = new RemoteCozy(COZY_URL)
+    // FIXME: Temporary hack to make cozy-client-js think it has OAuth tokens
+    remoteCozy._authstate = 3
+    remoteCozy._authcreds = Promise.resolve({
```

Comment:
I'm not sure to understand this one... Since `remoteCozy` is a `RemoteCozy` instance (and not a `cozy-client-js.Cozy` one), why do we need to do this?